source/documentation/transaction.html.md

```diff
@@ -136,8 +136,11 @@ val fResult = DB localTx { implicit s =>
 ### #Working with IO monads
```

Comment:
I overlooked this. The `#` in the title is unnecessary
```suggestion
### Working with IO monads
```

Reply:
Fixed. The other sections had leading `#`, too, and were fixed as well.